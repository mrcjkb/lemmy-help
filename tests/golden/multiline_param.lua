local U = {}

---Trigger a rebuild of one or more projects.
---@param opts table|nil optional configuration options:
---  * {select_mode} (JdtProjectSelectMode) Show prompt
---     to select projects or select all. Defaults
---     to 'prompt'
---
---  * {full_build} (boolean) full rebuild or
---     incremental build. Defaults to true (full build)
---@param reserverd table|nil reserved for the future use
---@return boolean
function U.multi_line(opts, reserverd)
    print(vim.inspect(opts), vim.inspect(reserverd))

    return true
end

---Multiline but missing description
---@param n number
---This is a special
---
---number
---@param m number
---And this is also
---
---@return number
function U.missing_desc(n, m)
    return n + m
end

return U
