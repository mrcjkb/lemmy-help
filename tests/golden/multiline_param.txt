U.multi_line({opts}, {reserverd})                                 *U.multi_line*
    Trigger a rebuild of one or more projects.

    Parameters: ~
        {opts}       (table|nil)  optional configuration options:
                                    * {select_mode} (JdtProjectSelectMode) Show prompt
                                       to select projects or select all. Defaults
                                       to 'prompt'

                                    * {full_build} (boolean) full rebuild or
                                       incremental build. Defaults to true (full build)
        {reserverd}  (table|nil)  reserved for the future use

    Returns: ~
        (boolean)


U.missing_desc({n}, {m})                                        *U.missing_desc*
    Multiline but missing description

    Parameters: ~
        {n}  (number)  This is a special

                       number
        {m}  (number)  And this is also


    Returns: ~
        (number)



